lib/datadog/ci/configuration/settings.rb

```diff
@@ -21,9 +21,15 @@ def self.add_settings!(base)
               end
 
               # DEV: Alias to Datadog::Tracing::Contrib::Extensions::Configuration::Settings#instrument.
-              # DEV: Should be removed when `c.ci.instrument` namespacing is complete.
+              # DEV: Should be removed when CI implement its own `c.ci.instrument`.
               define_method(:instrument) do |integration_name, options = {}, &block|
-                Datadog.configuration.send(:instrument, integration_name, options, &block)
+                Datadog.configuration.tracing.instrument(integration_name, options, &block)
```

Comment:
Does that mean `c.tracing.instrument :cucumber` and `c.ci.instrument :rails` both work?

Reply:
Technically yes, but its not really supposed to. Would have to add white/blacklist otherwise, to which I'm not really inclined to do. Will close this loophole in a future internal refactor.